presto-main/src/main/java/io/prestosql/execution/scheduler/SimpleNodeSelector.java

```diff
@@ -48,12 +48,12 @@
     private static final Logger log = Logger.get(SimpleNodeSelector.class);
 
     private final InternalNodeManager nodeManager;
-    private final NodeTaskMap nodeTaskMap;
-    private final boolean includeCoordinator;
-    private final AtomicReference<Supplier<NodeMap>> nodeMap;
-    private final int minCandidates;
-    private final int maxSplitsPerNode;
-    private final int maxPendingSplitsPerTask;
```

Comment:
We can use `private` itself right ?

Reply:
The access was changed from private to package-private so that these variables could be in OptimizedLocalNodeSelector which extends SimpleNodeSelector.